client/src/lib/stores/Regions.ts

```diff
@@ -13,7 +14,7 @@ export const RegionsStore = writable<Region[]>([]);
 */
 RegionsStore.subscribe((regions) => {
 	regions.forEach((region) => {
-		const winner = region.candidates.reduce(
+		let winner = region.candidates.reduce(
 			(prev, current) => (prev.count > current.count ? prev : current),
 			region.candidates[0]
 		);
```

Comment:
We could use a ternary here to reduce some work, and keep const correctness.

![image](https://user-images.githubusercontent.com/13600696/208251057-e77ed1dc-04bc-45ea-88da-11501c10f89f.png)

Reply:
Changed!